### PR TITLE
ci: sonar cloud exclude coverage checks as they are not accurate

### PR DESCRIPTION
**Pull request by sithlord48**
- Exclude all sources from the sonar cloud coverage checks they are not accurate at all.

### CODE REVIEW

**Review by nbolton — COMMENTED** [review]
If you want to dump the SC coverage, then you'll probably want to get rid of the badge on the readme.

**Comment by sithlord48** [comment]
> If you want to dump the SC coverage, then you'll probably want to get rid of the badge on the readme.

Fair enough let me update this pr

File: sonar-project.properties [hunk 1]
```diff
@@ -3,7 +3,7 @@ sonar.projectKey=deskflow_deskflow
 sonar.sources=scripts,src/cmd,src/gui,src/lib
 sonar.tests=src/test
 sonar.exclusions=subprojects/**,build/**
-sonar.coverage.exclusions=subprojects/**,scripts/**,src/test/**,src/gui/**
+sonar.coverage.exclusions=subprojects/**,scripts/**,src/test/**,src/gui/**,**
```
[the remark this answers is not in the document]
Comment (sithlord48) [reply sonar-project.properties:6]: how did i know this was going to be a suggestion lol

**Comment by nbolton** [comment]
I changed the gate. Do we still need this PR?

image

**Pull request closed without merging**